frontend/src/App.js

```diff
@@ -50,18 +50,16 @@ function App() {
           [
               {
                   command: command,
-                  args:
-                  [
-                      group: group,
-                      state: state 
+                  args:[
+                    args
                   ]
               }
           ]
       })
         }).then(response => {
           if(response.status === 200) {
             
-            console.log(`Post request using parameter: ${command}, ${group}, ${state}`);
+            console.log(`Post request using parameter: ${command}, and args: ${args}`);
```

Comment:
Since this is any array, printing the array like this will result in something like `Post request using parameter: ACTUATE_GROUP, and [object Object]`. You need to wrap it with a stringily if you want to print it: `JSON.stringify(args)`